src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -11,24 +11,34 @@
 import seedu.address.commons.exceptions.IllegalValueException;
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Person;
+import seedu.address.model.person.Seller;
 
 /**
  * An Immutable AddressBook that is serializable to JSON format.
  */
 @JsonRootName(value = "addressbook")
 class JsonSerializableAddressBook {
-
     public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
-
-    private final List<JsonAdaptedPerson> persons = new ArrayList<>();
+    private final List<JsonAdaptedSeller> sellers = new ArrayList<>();
+    // Maybe this can be changed to buyer, I am not too sure whether should we separate the sellers list and
+    // buyer list, or keep everything under a Person?
```

Comment:
maybe can keep both sellers & buyers under persons in Json?

Reply:
Yeap but I was thinking to seperate both buyer and seller could be better for clarity, but it really depends on what you guys want for this.